from je_auto_control.utils.je_auto_control_exception.exceptions import ImageNotFoundException

print(repr(ImageNotFoundException("test.png")))
print(repr(ImageNotFoundException("DW{PD}{DP}W{DOLD}PW{KDP{WKDP}WD][wpa][dwpa][dwaod()DW()Dikwa")))
